src/@vtex/gatsby-plugin-theme-ui/auth.ts

```diff
@@ -1,3 +1,3 @@
-import theme from '@vtex/gatsby-theme-store/components/Auth/Providers/theme'
+import { AuthProvidersTheme as theme } from '@vtex/store-ui'
 
 export default theme
```

Comment:
You do not need to rename the import, just export that.

```suggestion
import { AuthProvidersTheme } from '@vtex/store-ui'

export default AuthProvidersTheme
```